<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmAddWord</class>
 <widget class="QDialog" name="ConfirmAddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>确认添加的单词信息</string>
  </property>
  <widget class="QListWidget" name="trans_box">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>60</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="eng_label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>添加</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operate_btn</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>取消</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">operate_btn</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="operate_btn"/>
 </buttongroups>
</ui>
